www/templates/menu.html

```diff
@@ -34,7 +34,7 @@ <h1 class="title">PVTA</h1>
           <i class="icon ion-ios-location" style="margin-right: 10px"></i>
           Stops
         </ion-item>
-        <ion-item menu-close href="#/app/map">
+        <ion-item menu-close href="#/app/map/route">
```

Comment:
What good does this do again?  It's in the sidebar, so it's just gonna plot your current location, right?

For now, we can probably take this out.  I know you're planning on having a sidebar option that centers around you and shows you some close-by buses, which definitely will include this.  But for now? Nah.